cmd/queue/main.go

```diff
@@ -66,10 +67,10 @@ const (
 	drainSleepDuration = 30 * time.Second
 
 	// certPath is the path for the server certificate mounted by queue-proxy.
-	certPath = queue.CertDirectory + "/tls.crt"
+	certPath = queue.CertDirectory + "/" + certificates.SecretCertKey
```

Comment:
I guess the next step is to use the mounted secret to enable tls at the queue proxy internal servers.